presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -196,6 +196,136 @@ public void testCreatePartitionedTableAs(StorageFormat storageFormat)
         query(format("DROP TABLE %s", tableName));
     }
 
+    @Test(groups = STORAGE_FORMATS)
```

Comment:
The rest of the tests in this file seem to be for all formats. Let's instead create a new class named something like `TestHiveCsvStorageFormat`